observation/store.go

```diff
@@ -31,15 +32,19 @@ func NewStore(dBConnection DBConnection) *Store {
 func (store *Store) GetCSVRows(filter *Filter, limit *int) (CSVRowReader, error) {
```

Comment:
Do mind expanding the godoc to include what happens with no dimensions?

Reply:
👍